### ISSUE TITLE
Launch in background

### ISSUE THREAD

**Issue by mightyiam** [karma-runner/karma-chrome-launcher#27]
I know this is out of scope but out of interest...

I didn't find a way to specify opening the browser in the background. Stealing focus from the terminal window is just unwanted, for most cases, I guess.

I think it should be default.

There's a `flags` thing. I read [the manual](http://manpages.ubuntu.com/manpages/trusty/en/man1/chromium-browser.1.html). There's nothing. Perhaps a trick with `--display`?

It points to [this](https://developer.gnome.org/gtk3/stable/gtk-running.html) and [this](https://developer.gnome.org/gtk3/stable/gtk-x11.html), which also don't seem to have what is required. Perhaps something with `--class`? 

This is relevant for all the other "real" browser launchers, as well.

**Comment by jimmiebtlr** [comment]
A possible solution on linux would also be

    DISPLAY=:1 meteor

which just opens on another display.

**Comment by Dignifiedquire** [comment]
I totally agree that this would be nice, but I still have to see a way of making this actually possible, cross platform & cross browser :/

**Comment by aldendaniels** [comment]
+1

**Comment by chrisspurgeon** [comment]
Perhaps a related thing... when karma launches chrome that new chrome window seems to be the place where all new browser tabs appear. So for instance if the karma browser window is up and running and I follow a web link in an email the web page always loads as a new tab in the karma browser instance, as opposed to the normal chrome instance I already have running. Switching focus to the normal browser instance doesn't seem to have any effect. Any way to keep that from happening?

**Comment by bentrevor** [comment]
@chrisspurgeon my workaround was to start using Chromium as my default browser, and just use Chrome for karma.

**Comment by chrisspurgeon** [comment]
Ooh, great idea!

**Comment by markelog** [comment]
Just crossed my fingers - https://code.google.com/p/chromium/issues/detail?can=1&q=launch%20in%20background&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Cr%20Status%20Owner%20Summary%20OS%20Modified&id=534047&thanks=534047&ts=1442756400

**Comment by jstejada** [comment]
+1

**Comment by chadcallahan** [comment]
+1

**Comment by belveryin** [comment]
+1

**Comment by mspoerer** [comment]
+1

**Comment by sherzinger** [comment]
+1

**Comment by jmrowka-cyber** [comment]
+1

**Comment by shooksm** [comment]
Looks like running Chrome headless is just around the corner: https://www.chromestatus.com/features/5678767817097216

**Comment by antonybudianto** [comment]
https://developers.google.com/web/updates/2017/04/headless-chrome

**Comment by rogeriopvl** [comment]
Just a heads up, on OSX, headless chrome does behave completely like a background app. If you have a chrome already running, its window will be focused when you start headless mode.

Can anyone confirm this behaviour in other operating systems?

**Comment by captainjackrana** [comment]
@rogeriopvl headless chrome steals focus on OSX.. the desirable thing to have is what PhantomJS does, run completely silent in the background..
Are there any flags that can be passed as chrome options to change this behaviour?

**Comment by rogeriopvl** [comment]
@captainjackrana exactly. That is a known problem. I'm not aware of any flag that changes that behaviour. But it seems to me that it is OSX causing it.
